### PR TITLE
Run pronto in travis 

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/630/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1459/ : ABORTED in 17s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1720/ : FAILURE in 1m 07s

File: lib/gooddata.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ module GoodData
 end
 
 # Modules
-require_relative 'gooddata/bricks/bricks'
+ require_relative 'gooddata/bricks/bricks'
```
Comment (yenkins) [inline lib/gooddata.rb]: warning: Style/IndentationConsistency: Inconsistent indentation detected. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/631/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1460/ : FAILURE in 1m 04s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1721/ : FAILURE in 1m 02s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/632/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1461/ : FAILURE in 1m 02s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1722/ : FAILURE in 59s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/634/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1463/ : FAILURE in 1m 08s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1724/ : FAILURE in 1m 07s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/635/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1464/ : SUCCESS in 1m 02s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1725/ : FAILURE in 1m 03s

**Comment by panjan** [comment]
Pronto constantly gets 403 from GitHub API when creating comments. I used auth token of our service account @frfna. It fails in Travis but works locally.

`bundler: failed to load command: pronto (/home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/bin/pronto)
Octokit::TooManyRequests: GET https://api.github.com/repos/gooddata/gooddata-ruby/pulls?per_page=100: 403 - API rate limit exceeded for 35.188.73.34. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) // See: https://developer.github.com/v3/#rate-limiting
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/octokit-4.12.0/lib/octokit/response/raise_error.rb:16:in `on_complete'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/faraday-0.15.3/lib/faraday/response.rb:9:in `block in call'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/faraday-0.15.3/lib/faraday/response.rb:61:in `on_complete'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/faraday-0.15.3/lib/faraday/response.rb:8:in `call'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/octokit-4.12.0/lib/octokit/middleware/follow_redirects.rb:73:in `perform_with_redirection'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/octokit-4.12.0/lib/octokit/middleware/follow_redirects.rb:61:in `call'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/faraday-0.15.3/lib/faraday/request/retry.rb:128:in `call'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/faraday-0.15.3/lib/faraday/rack_builder.rb:143:in `build_response'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/faraday-0.15.3/lib/faraday/connection.rb:387:in `run_request'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/faraday-0.15.3/lib/faraday/connection.rb:138:in `get'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/sawyer-0.8.1/lib/sawyer/agent.rb:94:in `call'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/octokit-4.12.0/lib/octokit/connection.rb:156:in `request'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/octokit-4.12.0/lib/octokit/connection.rb:84:in `paginate'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/octokit-4.12.0/lib/octokit/client/pull_requests.rb:20:in `pull_requests'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto/github.rb:103:in `pull_requests'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto/github.rb:92:in `pull'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto/github.rb:83:in `pull_id'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto/github.rb:4:in `pull_comments'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto/formatter/pull_request_formatter.rb:6:in `existing_comments'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto/formatter/git_formatter.rb:6:in `format'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto.rb:67:in `block in run'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto.rb:66:in `each'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto.rb:66:in `run'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto/cli.rb:61:in `block in run'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto/cli.rb:60:in `chdir'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/lib/pronto/cli.rb:60:in `run'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/thor-0.19.4/lib/thor/command.rb:27:in `run'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/thor-0.19.4/lib/thor/invocation.rb:126:in `invoke_command'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/thor-0.19.4/lib/thor.rb:369:in `dispatch'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/thor-0.19.4/lib/thor/base.rb:444:in `start'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/gems/pronto-0.9.5/bin/pronto:6:in `<top (required)>'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/bin/pronto:23:in `load'
  /home/travis/build/gooddata/gooddata-ruby/vendor/bundle/ruby/2.2.0/bin/pronto:23:in `<top (required)>'`

**Pull request closed without merging**